txnpool/proc/txnpool_worker.go

```diff
@@ -62,6 +85,8 @@ func NewTxPoolWoker(s *TXPoolServer) *txPoolWorker {
 	worker.rspCh = make(chan *types.CheckResponse, tc.MAX_PENDING_TXN)
 	worker.stopCh = make(chan bool)
 	worker.server = s
+	worker.stateless = stateless.NewValidatorPool(2)
```

Comment:
May be worker can be more than 2? workerpool will shrink when idle in a specified period.